ckanext/dgu/schema.py

```diff
@@ -141,6 +141,8 @@
 
 update_frequency_options = ['never', 'discontinued', 'annual', 'quarterly', 'monthly']
 
+publication_delay_options = ['minimal', 'good', 'reasonable', 'extreme']
```

Comment:
I'm not even sure if/where this is being used. I was just copying update_frequency_options.

The front end should be showing the longer descriptions of these options from the form. Only the API will show these words.

Yes, I agree about using non-judgemental words instead.